contentprovider.go

```diff
@@ -835,7 +835,7 @@ func scoreSymbolKind(language string, filename []byte, sym []byte, kind ctags.Sy
 		switch kind {
 		case ctags.Class: // classes
 			factor = 10
-		case ctags.Function: // function definitions
+		case ctags.Function, ctags.Method: // function definitions
```

Comment:
Enabling these tests caught a bug in our Python logic: universal-ctags always emits `function`, but scip-ctags distinguishes between `function` and `method`.